src/game.js

```diff
@@ -374,7 +378,7 @@ class Game {
          * The 0th element is `game.player1`'s side of the board,
          * and the 1th element is `game.player2`'s side of the board.
          * 
-         * @type {[[Card], [Card]]}
+         * @type {Card[][]}
          */
         this.board = [[], []];
```

Comment:
Maybe have this be type: {0: [], 1: []}. It wouldn't break anything, but would make it better.

Reply:
This would be helpful, but should be done in another pr. #260